src/components/MovieDetailsPage/DetailsComponents/Duration.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { MovieInfoDetails } from '../../../types';
+
+type DurationProps = {
+  chosenMovieDetails: MovieInfoDetails
+};
+
+export const Duration: FC<DurationProps> = (props) => {
+  const { chosenMovieDetails } = props;
+  const { duration } = chosenMovieDetails;
+
+  return (
+    <p className="mt-2 text-danger fs-5">
+      <strong id="movies-counter" className="fw-bold">
+        {duration.toString()}
+      </strong>
+      {' '}
+      min
```

Comment:
let's combine it `{' min'}`

Reply:
Thanks. Fixed it in commit d0df8953a6498f884fc8f239cf86721b7a4c4d9a